hardware_enablement/psap-node-feature-discovery-operator.adoc

```diff
@@ -12,7 +12,16 @@ include::modules/psap-node-feature-discovery-operator.adoc[leveloffset=+1]
 
 include::modules/psap-installing-node-feature-discovery-operator.adoc[leveloffset=+1]
 
-include::modules/psap-using-node-feature-discovery-operator.adoc[leveloffset=+1]
+[id="psap-node-feature-discovery-operator-using-the-operator"]
+== Using the Node Feature Discovery Operator
+
+The Node Feature Discovery (NFD) Operator orchestrates all resources needed to run the Node-Feature-Discovery daemon set by watching for a `NodeFeatureDiscovery` custom resource (CR). Based on the `NodeFeatureDiscovery` CR, the Operator creates the operand (NFD) components in the desired namespace. You can edit the CR to use another namespace, image, image pull policy, and `nfd-worker-conf` config map, among other options.
```

Comment:
```suggestion
The Node Feature Discovery (NFD) Operator orchestrates all resources needed to run the Node-Feature-Discovery daemon set by watching for a `NodeFeatureDiscovery` custom resource (CR). Based on the `NodeFeatureDiscovery` CR, the Operator creates the operand (NFD) components in the selected namespace. You can edit the CR to use another namespace, image, image pull policy, and `nfd-worker-conf` config map, among other options.
```
IBM Style word usage warns against using the term "desire" or "desired" in technical documentation. I suggested "selected" here, but feel free to replace that if it's not quite right.